packages/calling/src/CallHistory/callHistoryFixtures.ts

```diff
@@ -325,6 +325,46 @@ const WEBEX_CALL_SESSION = {
   correlationIds: ['58ea6cd9-852b-4a77-957f-e704c8b0e63e'],
 };
 
+const SPARK_CALL_SESSION = {
+  id: '2ce4b9e0-7ee7-12a1-4a94-df524443b520',
+  durationSecs: 0,
+  self: {
+    id: '0fea4a63-4e27-46ee-99c3-2472cb12bf68',
+    name: 'Priya Kesari',
```

Comment:
Please change these names and put in test names. Applies in other parts of the fixture as well

Reply:
done